packages/kbn-unified-data-table/src/components/data_table.test.tsx

```diff
@@ -1144,9 +1148,9 @@ describe('UnifiedDataTable', () => {
     it(
       'should not allow comparison if comparison mode is disabled',
       async () => {
-        await renderDataTable({ enableComparisonMode: false });
-        selectDocument(esHitsMock[0]);
-        selectDocument(esHitsMock[1]);
+        renderDataTable({ enableComparisonMode: false });
```

Comment:
```suggestion
        await renderDataTable({ enableComparisonMode: false });
```
This one too

Reply:
Updated in [bf0438c](https://github.com/elastic/kibana/pull/189949/commits/bf0438cd67c88645fe112f6011ce9f182d1812bb).